jobs/cloud_controller_ng/templates/shutdown_drain.rb.erb

```diff
@@ -6,8 +6,6 @@ $LOAD_PATH.unshift('/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/l
 require 'cloud_controller/drain'
 
 @drain = VCAP::CloudController::Drain.new('/var/vcap/sys/log/cloud_controller_ng')
-
-@drain.shutdown_cc_uploader('/var/vcap/sys/run/bpm/cc_uploader/cc_uploader.pid')
```

Comment:
This change orphans some code in CC https://github.com/cloudfoundry/cloud_controller_ng/blob/b922faf661a4203a0089a2a1c80c212c47e75c9c/lib/cloud_controller/drain.rb#L61

I'm fine with this for the short term, but we probably want a PR reverting the CC changes also.